DotNet/bmp-endpoint.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text;
+
+using (var httpClient = new HttpClient { BaseAddress = new Uri("https://api.pdfrest.com") })
+{
+    using (var request = new HttpRequestMessage(HttpMethod.Post, "bmp"))
+    {
+        request.Headers.TryAddWithoutValidation("Api-Key", "XXXXXXXXXXXX");
```

Comment:
Just a small thing, but these `XXXXXXXXXXXX` could be replaced with `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`